deploy/infra/aws/terraform/cloud-native/cluster-infra/input.tf

```diff
@@ -233,3 +233,51 @@ variable "rds_db_port" {
   default     = 5432
   description = "The port on which the RDS instance accepts connections"
 }
+
+###################################################################
+#               Cloud-pipeline script deployment
+###################################################################
+
+variable "deployment_id" {
+  type        = string
+  description = "Deployment Name"
+}
+
+variable "elb_public_subnet" {
+  type        = string
+  description = "Public subnet for Load Balancer"
+}
+
+variable "eipalloc" {
+  type        = string
+  description = "Ellastic IP id"
+}
+
+variable "ui_deployment_name" {
+  type        = string
+  description = "Deployment Name that will be shown as the browser tab name"
+}
+
+variable "cp_api_srv_host" {
```

Comment:
for each of this parameter please add to the description as follow:
`This DNS name should be created in advanced by user and will be used to configure Cloud-Pipeline internal services (f.e. edge service).`